src/fu-util-common.c

```diff
@@ -1210,17 +1210,20 @@ fu_util_device_to_string (FwupdDevice *dev, guint idt)
 		/* TRANSLATORS: hardware state, e.g. "pending" */
 		fu_common_string_append_kv (str, idt + 1, _("Update State"),
 					    fwupd_update_state_to_string (state));
+
+		if (state == FWUPD_UPDATE_STATE_SUCCESS) {
+			tmp = fwupd_device_get_update_message (dev);
+			if (tmp != NULL) {
+				/* TRANSLATORS: helpful messages from last update */
+				fu_common_string_append_kv (str, idt + 1, _("Update Message"), tmp);
+			}
+		}
```

Comment:
do we want it here, or in fu_util_display_current_message?

Reply:
Here I think, this is what will translate into `get-devices` output.

Reply:
R5ght, i think we probably want it in both places.

Reply:
No, I don't think so.  If the firmware fails we don't call `fu_util_display_current_message`.  So why check for success?

Reply:
Indeed, you're correct, my apologies.